==xxxxx== Memcheck, a memory error detector
==xxxxx== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==xxxxx== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==xxxxx== Command: python /home/avalassi/CORAL/CORAL_2_3_26/src/../x86_64-slc5-gcc43-dbg/tests/bin/test_PyCoral_ImportPyCoral.py
==xxxxx== Parent PID: xxxxx
==xxxxx== 
--xxxxx-- 
--xxxxx-- Valgrind options:
--xxxxx--    -v
--xxxxx--    --leak-check=full
--xxxxx--    --show-reachable=yes
--xxxxx--    --error-limit=no
--xxxxx--    --log-file=/home/avalassi/CORAL/CORAL_2_3_26/src/logs/qmtest/valgrind/x86_64-slc5-gcc43-dbg/valgrind.pycoral.importpycoral.txt
--xxxxx--    --suppressions=/home/avalassi/CORAL/CORAL_2_3_26/src/config/qmtest/valgrind.supp
--xxxxx--    --gen-suppressions=all
--xxxxx--    --num-callers=50
--xxxxx--    --track-origins=yes
--xxxxx-- Contents of /proc/version:
--xxxxx--   Linux version 2.6.18-308.20.1.el5 (mockbuild@lxdist03) (gcc version 4.1.2 20080704 (Red Hat 4.1.2-52)) #1 SMP Wed Nov 14 10:31:58 CET 2012
--xxxxx-- Arch and hwcaps: AMD64, amd64-sse3-cx16
--xxxxx-- Page sizes: currently 4096, max supported 4096
--xxxxx-- Valgrind library directory: /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Python/2.6.5p2/x86_64-slc5-gcc43-opt/bin/python (0x400000)
--xxxxx-- Reading syms from /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/memcheck-amd64-linux (0x38000000)
--xxxxx--    object doesn't have a dynamic symbol table
--xxxxx-- Reading syms from /lib64/ld-2.5.so (0x3026000000)
--xxxxx-- Reading suppressions file: /home/avalassi/CORAL/CORAL_2_3_26/src/config/qmtest/valgrind.supp
--xxxxx-- Reading suppressions file: /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/default.supp
==xxxxx== embedded gdbserver: reading from /tmp/vgdb-pipe-from-vgdb-to-xxxxx-by-avalassi-on-lxmrra5001.cern.ch
==xxxxx== embedded gdbserver: writing to   /tmp/vgdb-pipe-to-vgdb-from-xxxxx-by-avalassi-on-lxmrra5001.cern.ch
==xxxxx== embedded gdbserver: shared mem   /tmp/vgdb-pipe-shared-mem-vgdb-xxxxx-by-avalassi-on-lxmrra5001.cern.ch
==xxxxx== 
==xxxxx== TO CONTROL THIS PROCESS USING vgdb (which you probably
==xxxxx== don't want to do, unless you know exactly what you're doing,
==xxxxx== or are doing some strange experiment):
==xxxxx==   /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/../../bin/vgdb --pid=xxxxx ...command...
==xxxxx== 
==xxxxx== TO DEBUG THIS PROCESS USING GDB: start GDB like this
==xxxxx==   /path/to/gdb python
==xxxxx== and then give GDB the following command
==xxxxx==   target remote | /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/../../bin/vgdb --pid=xxxxx
==xxxxx== --pid is optional if only one valgrind process is running
==xxxxx== 
--xxxxx-- REDIR: 0x3026014730 (strlen) redirected to 0x38045f87 (vgPlain_amd64_linux_REDIR_FOR_strlen)
--xxxxx-- Reading syms from /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/vgpreload_core-amd64-linux.so (0x4803000)
--xxxxx-- Reading syms from /afs/cern.ch/sw/lcg/external/valgrind/3.7.0p1/x86_64-slc5-gcc43-opt/lib/valgrind/vgpreload_memcheck-amd64-linux.so (0x4a05000)
--xxxxx-- REDIR: 0x3026014550 (index) redirected to 0x4a09690 (index)
--xxxxx-- REDIR: 0x3026014700 (strcmp) redirected to 0x4a09ff0 (strcmp)
--xxxxx-- Reading syms from /lib64/libpthread-2.5.so (0x3027400000)
--xxxxx-- Reading syms from /lib64/libdl-2.5.so (0x3027000000)
--xxxxx-- Reading syms from /lib64/libutil-2.5.so (0x302fc00000)
--xxxxx-- Reading syms from /lib64/libm-2.5.so (0x3026800000)
--xxxxx-- Reading syms from /lib64/libc-2.5.so (0x3026400000)
--xxxxx-- REDIR: 0x3026479b80 (memset) redirected to 0x4a0b550 (memset)
--xxxxx-- REDIR: 0x302647afc0 (memcpy) redirected to 0x4a0a610 (memcpy)
--xxxxx-- REDIR: 0x3026478cc0 (rindex) redirected to 0x4a09480 (rindex)
--xxxxx-- REDIR: 0x30264788d0 (strlen) redirected to 0x4a09a10 (strlen)
--xxxxx-- REDIR: 0x3026478b50 (strncmp) redirected to 0x4a09e50 (strncmp)
--xxxxx-- REDIR: 0x3026473b40 (malloc) redirected to 0x4a08689 (malloc)
--xxxxx-- REDIR: 0x3026474060 (realloc) redirected to 0x4a0873a (realloc)
--xxxxx-- REDIR: 0x30264715f0 (free) redirected to 0x4a08299 (free)
--xxxxx-- REDIR: 0x3026479090 (strstr) redirected to 0x4a0ba70 (strstr)
--xxxxx-- REDIR: 0x30264793c0 (memchr) redirected to 0x4a0a0b0 (memchr)
--xxxxx-- REDIR: 0x302647bc20 (rawmemchr) redirected to 0x4a0b870 (rawmemchr)
--xxxxx-- REDIR: 0x30264781a0 (index) redirected to 0x4a09550 (index)
--xxxxx-- REDIR: 0x30264789c0 (strnlen) redirected to 0x4a09990 (strnlen)
--xxxxx-- REDIR: 0x302647a6b0 (mempcpy) redirected to 0x4a0b8b0 (mempcpy)
--xxxxx-- REDIR: 0x3026478c10 (strncpy) redirected to 0x4a09bf0 (strncpy)
--xxxxx-- REDIR: 0x3026478390 (strcpy) redirected to 0x4a09a50 (strcpy)
--xxxxx-- REDIR: 0x3026477fe0 (strcat) redirected to 0x4a096d0 (strcat)
--xxxxx-- REDIR: 0x3026478ab0 (strncat) redirected to 0x4a09890 (strncat)
--xxxxx-- REDIR: 0x3026478350 (strcmp) redirected to 0x4a09f30 (strcmp)
--xxxxx-- REDIR: 0x30264799e0 (memmove) redirected to 0x4a0b5f0 (memmove)
--xxxxx-- REDIR: 0x302647acc0 (stpcpy) redirected to 0x4a0b210 (stpcpy)
--xxxxx-- REDIR: 0x302647bd70 (strchrnul) redirected to 0x4a0b840 (strchrnul)
--xxxxx-- REDIR: 0x3026473760 (calloc) redirected to 0x4a0794e (calloc)
--xxxxx-- Reading syms from /home/avalassi/CORAL/CORAL_2_3_26/x86_64-slc5-gcc43-dbg/lib/liblcg_PyCoral.so (0x85f8000)
--xxxxx-- Reading syms from /home/avalassi/CORAL/CORAL_2_3_26/x86_64-slc5-gcc43-dbg/lib/liblcg_CoralCommon.so (0x866d000)
--xxxxx-- Reading syms from /home/avalassi/CORAL/CORAL_2_3_26/x86_64-slc5-gcc43-dbg/lib/liblcg_CoralKernel.so (0x86e3000)
--xxxxx-- Reading syms from /home/avalassi/CORAL/CORAL_2_3_26/x86_64-slc5-gcc43-dbg/lib/liblcg_RelationalAccess.so (0x8721000)
--xxxxx-- Reading syms from /home/avalassi/CORAL/CORAL_2_3_26/x86_64-slc5-gcc43-dbg/lib/liblcg_CoralBase.so (0x8762000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Boost/1.48.0_python2.6/x86_64-slc5-gcc43-opt/lib/libboost_filesystem-gcc43-mt-1_48.so.1.48.0 (0x87d7000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Boost/1.48.0_python2.6/x86_64-slc5-gcc43-opt/lib/libboost_date_time-gcc43-mt-1_48.so.1.48.0 (0x89f5000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Boost/1.48.0_python2.6/x86_64-slc5-gcc43-opt/lib/libboost_system-gcc43-mt-1_48.so.1.48.0 (0x8c05000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Boost/1.48.0_python2.6/x86_64-slc5-gcc43-opt/lib/libboost_thread-gcc43-mt-1_48.so.1.48.0 (0x8e08000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/Python/2.6.5p2/x86_64-slc5-gcc43-opt/lib/libpython2.6.so.1.0 (0x9021000)
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/gcc/4.3.6/x86_64-slc5/lib64/libstdc++.so.6.0.10 (0x93da000)
--xxxxx--    object doesn't have a symbol table
--xxxxx-- Reading syms from /cvmfs/sft.cern.ch/lcg/external/gcc/4.3.6/x86_64-slc5/lib64/libgcc_s.so.1 (0x96e0000)
--xxxxx--    object doesn't have a symbol table
--xxxxx-- Reading syms from /lib64/librt-2.5.so (0x3027800000)
--xxxxx-- REDIR: 0x949a950 (operator new(unsigned long)) redirected to 0x4a08fa2 (operator new(unsigned long))
--xxxxx-- REDIR: 0x94991a0 (operator delete(void*)) redirected to 0x4a07fab (operator delete(void*))
==xxxxx== 
==xxxxx== HEAP SUMMARY:
==xxxxx==     in use at exit: 1,309,803 bytes in 538 blocks
==xxxxx==   total heap usage: 3,490 allocs, 2,952 frees, 3,167,370 bytes allocated
==xxxxx== 
==xxxxx== Searching for pointers to 538 not-freed blocks
==xxxxx== Checked 2,498,864 bytes
==xxxxx== 
==xxxxx== LEAK SUMMARY:
==xxxxx==    definitely lost: 0 bytes in 0 blocks
==xxxxx==    indirectly lost: 0 bytes in 0 blocks
==xxxxx==      possibly lost: 0 bytes in 0 blocks
==xxxxx==    still reachable: 0 bytes in 0 blocks
==xxxxx==         suppressed: 1,309,803 bytes in 538 blocks
==xxxxx== 
==xxxxx== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 702 from 63)
--xxxxx-- 
--xxxxx-- used_suppression:     33 <python26_Py_malloc_LEAK>
--xxxxx-- used_suppression:     40 <python26_PyImport_malloc_LEAK>
--xxxxx-- used_suppression:     10 <python26_PyImport_dlopen_LEAK>
--xxxxx-- used_suppression:    100 <python26_Py_dictresize_malloc_LEAK>
--xxxxx-- used_suppression:      1 <python26_Py_PyInit_malloc_LEAK>
--xxxxx-- used_suppression:    162 <python26_PyObject_GC_Malloc_malloc_LEAK>
--xxxxx-- used_suppression:      1 <python26_Py_calculatepath_malloc_LEAK>
--xxxxx-- used_suppression:      3 <python26_Py_realloc_LEAK>
--xxxxx-- used_suppression:     23 <python26_Py_realloc_malloc_LEAK>
--xxxxx-- used_suppression:      1 <coral_PyCoral_import_LEAK>
--xxxxx-- used_suppression:      1 <python26_Py_strdup_malloc_LEAK>
--xxxxx-- used_suppression:    630 <python26_PyObject_Free_addr4>
--xxxxx-- used_suppression:      7 <python26_PyObject_Free_val8>
--xxxxx-- used_suppression:      7 <python26_PyObject_Free_cond>
--xxxxx-- used_suppression:     45 dl-hack3
==xxxxx== 
==xxxxx== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 702 from 63)
